blockchain/ens.ts

```
@@ -8,7 +8,7 @@ export function resolveENSName$(context$: Observable<Context>, address: string)
   return context$.pipe(
     switchMap(async (context) => {
       const provider = new ethers.providers.JsonRpcProvider(context.infuraUrl)
-      return await provider.lookupAddress(address)
+      return await provider.lookupAddress(address).catch(() => null)
```

so the intention is return null on error ?

---

yes, or empty string, or any nullish value

---

It could be useful to only catch (and possibly log) when the network doesn't support ENS, otherwise we are failing silently in all other error situations too.